crates/telemetry-subscribers/Cargo.toml

```diff
@@ -26,7 +26,7 @@ tokio = { workspace = true, features = ["full"] }
 futures.workspace = true
 clap.workspace = true
 bytes.workspace = true
-bytes-varint = { version = "1" }
+bytes-varint = { version = "=1.0.3" }
```

Comment:
This is needed to use this exact version, as the latest version breaks the project compilation atm

Reply:
1. Consider turning this into an issue: this will need to be resolved one way or another, even if this PR is not the place for it.
2. Instead of _fixing_ the version at exactly `=1.0.3`, consider using something like `<1.1.0` instead. This has a couple upsides:
    - If a `1.0.4` patch is released, you will still automatically make use of it,
    - You now know precisely which version is introducing the bug/problem (it is `1.1.0`, not `1.0.4` or something like that), even when new patches are put in place.